components/team_sidebar/team_sidebar.tsx

```diff
@@ -259,6 +259,14 @@ export default class TeamSidebar extends React.PureComponent<Props, State> {
 
         const joinableTeams = [];
 
+        const plusIcon = (
+            <i
+                className='icon icon-plus'
+                role={'img'}
+                aria-label={'Plus Icon'}
```

Comment:
we localize the aria labels too, can we do that here

Reply:
Done. I also added localization in `team_icon.tsx`.